wiki/Help_Centre/Restricted_Mode/en.md

```diff
@@ -0,0 +1,60 @@
+- **Wait 24 hours.** Some restrictions are the results of automated processes. As we check every new restriction daily, please wait 24 hours - if your restriction was erroneous, it will be lifted within that time.
+- **Contact our support team.** If your account remains restricted after that time, please send an email to [accounts@ppy.sh](mailto:accounts@ppy.sh) **from the email address attached to your account** to ask for details on your restriction. Make sure you include your username, too!
+
+If your account has become restricted for breaking the [rules](/wiki/rules), a strict cooldown of three months will be enforced before you are allowed to send an appeal.
+
+If your original account is not restricted, you may log into it and continue playing normally. but make sure not to log into your multi-accounts from then on, as doing so may lead to further punishment.
+
+If you have recently been granted an appeal and your account becomes restricted again after playing, do not panic! Automated systems may have falsely flagged your account, but you will be unrestricted again within 24 hours if this is found to be the case. Feel free to send us an email or a tweet to catch our attention and be unrestricted faster!
+
+## Why am I restricted?
+
+There are several reasons why an account might be restricted.
+- **It is not your first account.** Multi-accounting is against the rules. Any accounts other than a player's first will become permanently restricted, but their original one will in most cases be left open and the user is encouraged to continue playing on it. If the multi-accounting is severe or abusive, a silence may be applied to their original account as a warning to cease creating new ones, and a restriction will be placed on the main account if the multi-accounting behaviour continues. If your account is restricted and you know you own other ones, please do not continue accessing the restricted accounts you have and stick to playing on your original one. If you have trouble accessing or verifying it, please send us an email at [accounts@ppy.sh](mailto:accounts@ppy.sh) so that we can help you recover access to it.
+- **You have allowed someone to play on your account.** Account sharing is also against the rules. If you allow someone to use your account, it will be restricted, and the player who used it will likely receive a silence on their account or be restricted themselves. If this happened accidentally, please let us know as soon as possible by sending an email to [accounts@ppy.sh](mailto:accounts@ppy.sh).
+- **You cheated.** Using a third-party program, macros, or any form of unfair advantage will result in an immediate restriction.
+- **Other cases.** The accounts team reserves the right to restrict an account for cases that are not covered by the three above. This generally includes investigating allegations of cheating or punishing excessive bad behaviour on the forums or chats.
```

Comment:
the “allegations of cheating” feels a bit weird here. i don’t think it’s required to state this as it’s covered in the previous point?

Reply:
I'll remove the last sentence, I guess the first statement is sufficient